system.c

```diff
@@ -265,7 +266,35 @@ uint8_t system_execute_line(char *line)
           if ( line[++char_counter] != 0 ) { return(STATUS_INVALID_STATEMENT); }
           else { report_ngc_parameters(); }
           break;
-
+        case 'A': // Probe: Specify sensor, axis and specified directionX,
+          char_counter++;
+          uint8_t axis = line[char_counter];
+          axis = get_axis_idx(axis);
+          //Axis
+          if ((axis == N_AXIS) || (line[++char_counter] != ','))
```

Comment:
Do any other gcode commands include commas?  I'd rather see it be idiomatic if possible